src/app/components/core/logs/system-log/system-log.component.ts

```diff
@@ -256,9 +254,14 @@ export class SystemLogComponent implements OnInit, OnDestroy {
     let sourceSchedule: any = [...this.scheduleData].find((sch: any) => sch.name === this.source)
     if (sourceSchedule.processName === 'south_c') {
       this.routePath = '/south';
+      this.showConfigButton = true;
     }
-    else {
+    else if (sourceSchedule.processName === 'north_c') {
       this.routePath = '/north';
+      this.showConfigButton = true;
+    }
+    else {
+      this.showConfigButton = false;
```

Comment:
I assume in init we have default set to false

Reply:
Yes default value is `false`